src/Microsoft.AspNet.Mvc.Core/Rendering/Html/TemplateBuilder.cs

```diff
@@ -5,9 +5,9 @@
 using Microsoft.AspNet.Mvc.ModelBinding;
 using Microsoft.Framework.Internal;
 
-namespace Microsoft.AspNet.Mvc.Rendering
+namespace Microsoft.AspNet.Mvc.Rendering.Internal
```

Comment:
Eh, no, I don't think so. The common (and logical) pattern is to add a `.Internal` namespace in the namespace you're in. Otherwise you end up with a super weird hierarchy.